tests/autotuner/test_autotune_cache_v2.py

```
@@ -0,0 +1,251 @@
+    # Atomic publication leaves no temp files behind.
+    assert list(cache_root.glob("v2/*/entries/*.tmp")) == []
+
+
+def test_fresh_process_reuses_entry_without_profiling(cache_root, monkeypatch):
+    _tune_once(monkeypatch, times={0: 3.0, 1: 1.0, 2: 2.0})
+
+    # Simulate a new process: wipe all in-memory state.
+    reset_autotuner()
+    calls = _install_fake_profile(monkeypatch, times={0: 3.0, 1: 1.0, 2: 2.0})
+    with autotune(False, cache=True):
+        runner, tactic = AutoTuner.get().choose_one(
```

Ruff (0.15.20)

[warning] Unpacked variable `runner` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)